go.mod

```diff
@@ -14,33 +14,13 @@ require (
 	k8s.io/apimachinery v0.16.15
 	k8s.io/client-go v0.16.15
 	k8s.io/code-generator v0.18.3
```

Comment:
`k8s.io/code-generator` can also be downgraded to v1.16.15 by the way though it has no effect here :)

Reply:
Yeah. I will fix it